services/galley/src/Galley/API/Teams/Features.hs

```diff
@@ -428,6 +428,15 @@ instance SetFeatureConfig MlsMigrationConfig where
       $ throw MLSProtocolMismatch
     pure feat
 
-instance SetFeatureConfig EnforceFileDownloadLocationConfig
+instance SetFeatureConfig EnforceFileDownloadLocationConfig where
+  type
+    SetFeatureForTeamConstraints EnforceFileDownloadLocationConfig r =
+      (Member (Error TeamFeatureError) r)
+
+  prepareFeature _ feat = do
+    -- empty download location is not allowed
```

Comment:
```suggestion
    -- empty download location is not allowed
    -- this is consistent with all other features, and least surprising for clients
```

Reply:
i guess for the same reason this check should be added to the timestamps for `MlsMigrationConfig`?